Lib/test/test_abc.py

```diff
@@ -479,15 +494,23 @@ class ReceivesClassKwargs:
                 def __init_subclass__(cls, **kwargs):
                     super().__init_subclass__()
                     saved_kwargs.update(kwargs)
-            class Receiver(ReceivesClassKwargs, abc_ABC, x=1, y=2, z=3):
+            class Receiver(ReceivesClassKwargs, abc_ABC, x=1, y=2, __module__='some.module'):
                 pass
-            self.assertEqual(saved_kwargs, dict(x=1, y=2, z=3))
+            # __module__ passed via __init_subclass__ is honored over
+            # the value set via caller inspection
+            self.assertEqual(saved_kwargs, dict(x=1, y=2, __module__='some.module'))
     return TestLegacyAPI, TestABC, TestABCWithInitSubclass
 
+# types used for pickle test
+ABCToPickle_C = abc.ABCMeta('ABCToPickle_C', (), {'data': 123})
+ABCToPickle_Py = _py_abc.ABCMeta('ABCToPickle_Py', (), {'data': 123})
+
 TestLegacyAPI_Py, TestABC_Py, TestABCWithInitSubclass_Py = test_factory(abc.ABCMeta,
-                                                                        abc.get_cache_token)
+                                                                        abc.get_cache_token,
+                                                                        ABCToPickle_C)
 TestLegacyAPI_C, TestABC_C, TestABCWithInitSubclass_C = test_factory(_py_abc.ABCMeta,
```

Comment:
I noticed that the variables prefix do not match the module passed to the factory (they are flipped), is this a typo?